techniques/systemSettings/misc/rudderSNMP/1.0/rudderSNMP.st

```diff
@@ -46,10 +46,11 @@ bundle agent rudderSNMP
     !file_ensure_lines_present__etc_snmp_snmpd_conf_repaired::
       "reporting" usebundle => rudder_common_report("rudderSNMP", "result_na", "&TRACKINGKEY&", "SNMP service restart", "None", "No need to restart SNMP service");
 
-    package_present_${canon_pkg_name}_repaired::
-      "method_call" usebundle => file_enforce_content("/etc/snmp/snmpd.conf", "rocommunity public", "false");
-      "reporting" usebundle => rudder_common_reports_generic("rudderSNMP", "file_ensure_lines_present__etc_snmp_snmpd_conf", "&TRACKINGKEY&", "SNMP configuration file contains required community", "None", "Checking the configuration file for SNMP");
+      "method_call" usebundle => file_enforce_content("/etc/snmp/snmpd.conf", "rocommunity public", "false"),
+        ifvarclass => "package_present_${canon_pkg_name}_repaired";
```

Comment:
this line dupplicates what is done on line 24 - rocommunity public is already added there